bigbluebutton-html5/imports/api/breakouts-history/server/publishers.js

```diff
@@ -0,0 +1,41 @@
+import BreakoutsHistory from '/imports/api/breakouts-history';
+import { Meteor } from 'meteor/meteor';
+import { check } from 'meteor/check';
+import AuthTokenValidation, { ValidationStates } from '../../auth-token-validation';
+import Logger from '../../../startup/server/logger';
+import Meetings from '../../meetings';
+import Users from '../../users';
+
+const ROLE_MODERATOR = Meteor.settings.public.user.role_moderator;
+
+function breakoutsHistory() {
+  const tokenValidation = AuthTokenValidation.findOne({ connectionId: this.connection.id });
+
+  if (!tokenValidation || tokenValidation.validationStatus !== ValidationStates.VALIDATED) {
```

Comment:
Another area to review depending on when https://github.com/bigbluebutton/bigbluebutton/pull/13574 is merged